yt_dlp/extractor/cineverse.py

```diff
@@ -76,11 +104,30 @@ def _real_extract(self, url):
                 'You may be able to bypass it by using the /details/ page instead of the /watch/ page',
                 countries=smuggled_data.get('geo_countries'))
 
+        # there can be multiple age limits (e.g. PG-13 AND TV-14), so take first
+        age_limit = None
+        age_limits = traverse_obj(idetails, ('details', 'rating_code', {lambda x: x.split(', ')}))
+        if age_limits and len(age_limits) > 0:
+            age_limit = parse_age_limit(age_limits[0])
```

Comment:
i think the "proper" thing to do in cases like this is to use whatever rating is highest
"better safe than sorry"
```suggestion
        age_limit = None
        age_limits = traverse_obj(idetails, ('details', 'rating_code', {lambda x: x.split(', ')}))
        if age_limits and len(age_limits) > 0:
            age_limit = 0
            for rating in age_limits:
                age_limit = max(parse_age_limit(rating), age_limit)
```
i set `age_limit` to 0 so that we don't get an error trying to compare with `None`

Reply:
ah hold on, scratch that
```suggestion
        age_limit = None
        age_limits = traverse_obj(idetails, ('details', 'rating_cdode', {lambda x: x.split(', ')}, ..., {parse_age_limit}))
        if age_limits and len(age_limits) > 0:
            age_limit = max(age_limits)
```
we can do traverse_obj magic to run parse_age_limit on every rating in the list
and then just feed that list to `max()`

Reply:
We should also be able to use `..., {parse_age_limit}, all, {max}` to get the max for all:
```
age_limit = traverse_obj(idetails, ('details', 'rating_code', {lambda x: x.split(', ')}, ..., {parse_age_limit}, all, {max}))
```

Reply:
Good idea. Committed @Grub4K's suggestion and updated a unit test affected by the change.